### ISSUE TITLE
PDOStatement::getColumnMeta() crash/segfault

### ISSUE THREAD

**Issue by v-kigos** [Microsoft/msphpsql#224]
PHP call for $stmt->getColumnMeta($index) will crash in Windows, and produce
segmentation fault in Linux if index is out of the range (exceeds the number of columns).

Repro script:
```
<?php
/* Connect to an SQL server */
$conn = new PDO("sqlsrv:server=$server", $user, $password);  

/* Retrieve metadata for a column */
$stmt = $conn->query("select name from sys.databases");  
$metadata = $stmt->getColumnMeta(2);
var_dump($metadata);  
?>
```

**Comment by yitam** [comment]
It also crashes when prepare() is used instead of query() when the column is valid.

**Comment by v-kigos** [comment]
MSDN does not define return values for this case, 
see https://msdn.microsoft.com/en-us/library/ff628182(v=sql.105).aspx

PHP help suggests: Returns FALSE if the requested column does not exist in the result set, or if no result set exists, see http://php.net/manual/en/pdostatement.getcolumnmeta.php

**Comment by yitam** [comment]
Fixed by #261

**Comment by eaglebush** [comment]
My development system is Windows 7 SP1 and the PDO_SQLSRV I use is 4.1.4. The only problem I am having is PDOStatement::getColumnMeta() crash/segfault #224, fixed in 4.3.0. However, this version does not support Windows 7 SP1. Can't we have an update on PDO_SQLSRV 4.1.4 of this bug?

**Comment by yitam** [comment]
Yes @eaglebush , starting with v4.3.0 we stopped supporting Windows 7. That means we will no longer test our drivers in Windows 7 platforms, but it doesn't mean the drivers will break. Please feel free to try out v4.3.0 in your Windows 7 SP1.
